libminifi/include/utils/IntegrationTestUtils.h

```diff
@@ -0,0 +1,66 @@
+/**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <utility>
+#include <string>
+#include <vector>
+
+#include "../../../libminifi/test/TestBase.h"
```

Comment:
In case libminifi is added as "target_include_dir", this can be made nice and simple.

Reply:
Yes, most of the time it is, but now always. There other places includes happen like this, might worth the investigation why, but I don't think it is relevant for this PR.

Reply:
See the Jira created here:
https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=405&view=detail&selectedIssue=MINIFICPP-1311